tests/core/test_subdir_data.py

```diff
@@ -305,14 +352,3 @@ def test_metadata_cache_clearing(platform=OVERRIDE_PLATFORM):
         precs_b = tuple(sd_b.query("zlib"))
         assert fetcher.call_count == 2
         assert precs_b == precs_a
-
-
-# @pytest.mark.integration
-# class SubdirDataTests(TestCase):
-#
-#     def test_basic_subdir_data(self):
-#         channel = Channel("https://conda.anaconda.org/conda-test/linux-64")
-#         sd = SubdirData(channel)
-#         sd.load()
-#         print(sd._names_index.keys())
-#         assert 0
```

Comment:
🔥

Reply:
It's been commented out for a long time.